x-pack/plugins/security_solution/public/common/containers/cti/event_enrichment/use_investigation_enrichment.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useCallback, useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { isEmpty } from 'lodash';
+
+import { EventFields } from '../../../../../common/search_strategy/security_solution/cti';
+import { useAppToasts } from '../../../hooks/use_app_toasts';
+import { useKibana } from '../../../lib/kibana';
+import { inputsActions } from '../../../store/actions';
+import * as i18n from './translations';
+import { useEventEnrichment } from '.';
+
+export const QUERY_ID = 'investigation_time_enrichment';
+const noop = () => {};
+
+export const useInvestigationTimeEnrichment = (eventFields: EventFields) => {
+  const { addError } = useAppToasts();
+  const kibana = useKibana();
+  const dispatch = useDispatch();
+  const [{ from, to }, setRange] = useState({ from: 'now-30d', to: 'now' });
```

Comment:
why not hardcode `from` and `to` as `'now-30d'` and `'now'` in the cti constants file? and not use useState as `setRange` doesn't seem to be utilized?

Reply:
I moved the defaults to constants alongside the default index, but `setRange` is here as that's going to be an argument to the "no data, update lookback?" component.